app/controllers/spree/admin/intranet_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Spree
+  module Admin
+    class IntranetController < ResourceController
+      before_action :authorize_intranet_access
+
+      def index
+        @pages = Spree::Page.where(show_on_intranet: true)
```

Comment:
This `@pages` ivar isn't used in the view. We rely on the `@collection` instead.

```suggestion
```